wire/receiver_internal_test.go

```diff
@@ -39,7 +38,7 @@ func TestReceiver_Close(t *testing.T) {
 
 func TestReceiver_Next(t *testing.T) {
 	t.Parallel()
-	e := newRandomEnvelope(test.Prng(t), NewPingMsg())
```

Comment:
`newRandomEnvelope` generates an envelope with the specified message and randomized addresses. For instantiating a random address we need to have an address type that we can use. For the tests we typically use sim/wire.Address, but we can't use that here because it imports package wire and creates a circular dependency. In fact, any implementation of wire.Address would have this problem.

The only way we could run the tests with a concrete address type would be to implement one within the wire package, but since we don't really want that and for the tests it didn't seem crucial that the envelope addresses are set, I opted for removing the envelope addresses altogether and setting them to nil.